eodag/plugins/download/creodias_s3.py

```diff
@@ -62,3 +72,18 @@ def _get_bucket_names_and_prefixes(
                 self.get_product_bucket_name_and_prefix(product, s3_url)
             ]
         return bucket_names_and_prefixes
+
+    def presign_url(
```

Comment:
`presign_url` need to be a `Auth` plugin method.
To check if presigning URLs is supported, add a configuration parameter called `support_presign_url` in the `AwsAuth` plugin configuration and set the default value to `true`.